modules/gitops-using-argo-cd-instance-to-manage-cluster-scoped-resources.adoc

```diff
@@ -6,7 +6,7 @@
 [id="using-argo-cd-instance-to-manage-cluster-scoped-resources{context}"]
 = Using an Argo CD instance to manage cluster-scoped resources
 
-To manage cluster-scoped resources, update the existing `Subscription` object for the `gitops-title` Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
+To manage cluster-scoped resources, update the existing `Subscription` object for the `{gitops-title}` Operator and add the namespace of the Argo CD instance to the `ARGOCD_CLUSTER_CONFIG_NAMESPACES` environment variable in the `spec` section.
```

Comment:
could you add an example here? Spec doesn't really seem specific enough. 
Should look like:
```
spec:
  config:
    env:
      - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
        value: '<namespace-of-argocd-instance-you-want-cluster-scoped>'
```

Reply:
@reginapizza: Can you please take a look at L25 to L29? We have covered this already.

spec:
  config:
    env:
    - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
      value: openshift-gitops, <list of namespaces of cluster-scoped Argo CD instances>

Reply:
ah ok! Nevermind then, all good :)